docs/content/users/install/ddev-upgrade.md

```diff
@@ -0,0 +1,117 @@
+# Upgrading DDEV
+
+Installing and upgrading DDEV are nearly the same thing, because you're upgrading the `ddev` binary that talks with Docker. You can update this file like other software on your system, whether it’s with a package manager or traditional installer.
+
+=== "macOS"
+
+    ## macOS
+
+    ### Homebrew
+
+    ```bash
+    # Upgrade DDEV to the latest version
+    brew upgrade ddev
+    ```
+
+    ### Install Script
+
+    ```bash
+    # Download and run the script to replace the DDEV binary
+    curl -fsSL https://ddev.com/install.sh | bash
+    ```
+
+    ??? "Need a specific version?"
+        Use the `-s` argument to specify a specific stable or prerelease version:
+
+        ```bash
+        # Download and run the script to update to DDEV v1.21.4
+        curl -fsSL https://ddev.com/install.sh | bash -s v1.21.4
+        ```
+
+=== "Linux"
+
+    ## Linux
+
+    ### Debian/Ubuntu
+
+    ```bash
+    # Update package information and all packages including DDEV
+    sudo apt update && sudo apt upgrade
+    ```
+
+    ### Fedora, Red Hat, etc.
+
+    ```bash
+    # Upgrade the DDEV package
+    sudo dnf upgrade ddev
+    ```
+
+    ### Arch Linux
+
+    ```bash
+    # Upgrade the DDEV package
+    yay -Syu ddev-bin
+    ```
+
+=== "Windows"
+
+    ## Windows
+
+    ### WSL2 + Docker Install Script
```

Comment:
This section is not correct. For all WSL2, the upgrade technique is same as Debian/Ubuntu. Just open Ubuntu and `sudo apt update && sudo apt upgrade ddev` - no mucking about with PowerShell or anything.

Reply:
I’m not sure how I managed to get that so wrong!

> But the upgrade of ddev for all WSL2 Ubuntu installation methods is `sudo apt update && sudo apt upgrade`

Your comment above is slightly different. Which should it be?

1. `sudo apt update && sudo apt upgrade`
2. `sudo apt update && sudo apt upgrade ddev`

Reply:
So you’re recommending `sudo apt upgrade -y`—are you deliberately leaving out `sudo apt update &&`?

Or is the full and “final” answer `sudo apt update && sudo apt upgrade -y`?

Reply:
`sudo apt update && sudo apt upgrade -y` is the full and final answer, thanks.

Reply:
Fixed in d2a35731c2e639edf603b454425c00653a932524, which is refreshingly more concise!